keps/sig-network/4974-deprecate-endpoints/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Deprecate v1.Endpoints and Associated Controllers
+kep-number: 4974
+authors:
+  - "@danwinship"
+owning-sig: sig-network
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+  - sig-api-machinery
+status: implementable
+creation-date: 2024-11-21
+reviewers:
+  - "@aojea"
+  - "@robscott"
+  - "@thockin"
+approvers:
+  - "@aojea"
+  - "@thockin"
+see-also:
+  - "/keps/sig-network/0752-endpointslices"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: deprecated
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  deprecated: "v1.33"
```

Comment:
Maybe we can add your stages here so we can call out what work will need to be done over time.

Reply:
`kep.yaml` has required syntax, so we can't really (except as comments, but then it's just duplicating what's already in the README)

Reply:
Aww that’s a shame.